airbyte-integrations/connectors/source-azure-table/source_azure_table/spec.json

```diff
@@ -17,14 +17,14 @@
       "storage_access_key": {
         "title": "Access Key",
         "type": "string",
-        "description": "The access key to your storage account. Read more about access keys here - https://docs.microsoft.com/en-us/azure/storage/common/storage-account-keys-manage?tabs=azure-portal#view-account-access-keys",
```

Comment:
Please, update `airbyte/docs/integrations/sources/azure-table.md` and set this information to `md` file